lib/erlef_web/views/event_view.ex

```diff
@@ -39,6 +39,12 @@ defmodule ErlefWeb.EventView do
 
   def time(%DateTime{} = datetime), do: format_time(datetime)
 
+  def gcal_id(id) when id == nil, do: ""
```

Comment:
One thing we can do here is to remove the guard and pattern match in the function head. 

```
def foo(nil), do: ""
def foo(other), do: "foo-#{other}"
```